### ISSUE TITLE
Future of the library liquid and vdom

### ISSUE THREAD

**Issue by fleugzeug** [localvoid/liquid#12]
Hello,
First I thank you for your work.
My question is have you planned improvements to these two libraries ?

**Comment by localvoid** [comment]
Hi,
I am not sure what to do right now with this libraries. I've found a way to make them work way much faster, but it will require complete rewrite of vdom and liquid. I also don't like some parts of the API, like `final myComponent = vdom.componentFactory(MyComponent);`, but I am out of ideas how to implement it in a better way without any serious perf cost.
I also found that dart2js generated code is really slow in Firefox, even with `'--trust-type-annotations', '--trust-primitives'` flags, and `dart:html` abstraction has a ridiculous perf cost for things like assigning classes to dom elements.

The problem is that I don't use this library in any projects, so its future depends on how much free time I have to work on it. And I really don't want to be a maintainer of project like this, I would be way much happier if I could just write code for project like this.
